### ISSUE TITLE
Ctrl+Scroll while searching scrolls 2 tabs.

### ISSUE THREAD

**Issue by pjootr93** [PoE-Overlay-Community/PoE-Overlay-Community-Fork#137]
Ctrl+Scroll while searching scrolls twice. I think ctrl+scroll from macro works only when cursor is not over stash, because then standard game shortcut works, but when I use ctr+F and don't type enter it's scroll 2 tabs for 1 scroll.

**Comment by Bob343** [comment]
I don't think this is caused by overlay - with the scrolling disabled (and confirmed code isn't executing), it's still scrolling 2 tabs when on Ctrl+F in stash. I think this is just PoE
